covid-nlp/main.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        lemma_words = ["be", "man", "woman", "have", "do", "emergency", "epidemic", "outbreak", "crisis",
</pre><p>get a path to the list of lemmatized words as a second input.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>